src/utils/combineReducers.js

```diff
@@ -97,9 +98,16 @@ export default function combineReducers(reducers) {
     }
   });
 
+  if (sanityError) {
+    console.error(sanityError.message);
+  }
+
```

Comment:
`console.logs/errors` are a no-no in the library because they are normally uncatchable and noisy (and `console` sometimes doesn't exist). A thrown `Error` is best because that follows normal JS patterns.